test/functional/plugin/man_spec.lua

```diff
@@ -197,6 +197,19 @@ describe(':Man', function()
     matches('quit works!!', fn.system(args, { 'manpage contents' }))
   end)
 
+  it('q quits in "$MANPAGER mode" after jumping to a different tag (:Man!) #25943', function()
```

Comment:
this test is nearly a duplicate of the above. can you just add `+tag ls` to the above test , I don't think we need a new, separate test.